src/Umbraco.Web.UI/config/umbracoSettings.Release.config

```diff
@@ -1,113 +1,245 @@
-    internalRedirectPreservesTemplate="false" disableAlternativeTemplates="false" validateAlternativeTemplates="false" disableFindContentByIdPath="false"
-    umbracoApplicationUrl="">
-  </web.routing>
-
-</settings>
+<?xml version="1.0" encoding="utf-8" ?>
+<settings>
+
+  <!--
+    umbracoSettings.config configuration documentation can be found here:
+    https://our.umbraco.com/documentation/using-umbraco/config-files/umbracoSettings/
+    Many of the optional settings are not explicitly listed here
+    but can be found in the online documentation.
+    -->
+
+    <backOffice>
+        <tours enable="true"></tours>
+    </backOffice>
+
+  <content>
+
+    <errors>
+      <error404>1</error404>
+      <!--
+        The value for error pages can be:
+        * A content item's GUID ID      (example: 26C1D84F-C900-4D53-B167-E25CC489DAC8)
+        * An XPath statement            (example: //errorPages[@nodeName='My cool error']
+        * A content item's integer ID   (example: 1234)
+      -->
+      <!--
+        <error404>
+            <errorPage culture="default">26C1D84F-C900-4D53-B167-E25CC489DAC8</errorPage>
+            <errorPage culture="en-US">D820E120-6865-4D88-BFFE-48801A6AC375</errorPage>
+        </error404>
+       -->
+    </errors>
+
+    <notifications>
+      <!-- the email that should be used as from mail when umbraco sends a notification -->
+      <!-- you can add a display name to the email like this: <email>Your display name here &lt;your@email.here&gt;</email> -->
+      <email>your@email.here</email>
+    </notifications>
+
+    <!-- The html injected into a (x)html page if Umbraco is running in preview mode -->
+    <PreviewBadge>
+        <![CDATA[
+            <div id="umbracoPreviewBadge" class="umbraco-preview-badge">
+                <span class="umbraco-preview-badge__header">Preview mode</span>
+                <a href="{0}/preview/?id={2}" class="umbraco-preview-badge__a open">
+                    …
+                </a>
+                <a href="{0}/preview/end?redir={1}" class="umbraco-preview-badge__a end">
+                    <svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg"><title>Click to end</title><path fill="#fff" d="M5273.1 2400.1v-2c0-2.8-5-4-9.7-4s-9.7 1.3-9.7 4v2a7 7 0 002 4.9l5 4.9c.3.3.4.6.4 1v6.4c0 .4.2.7.6.8l2.9.9c.5.1 1-.2 1-.8v-7.2c0-.4.2-.7.4-1l5.1-5a7 7 0 002-4.9zm-9.7-.1c-4.8 0-7.4-1.3-7.5-1.8.1-.5 2.7-1.8 7.5-1.8s7.3 1.3 7.5 1.8c-.2.5-2.7 1.8-7.5 1.8z"/><path fill="#fff" d="M5268.4 2410.3c-.6 0-1 .4-1 1s.4 1 1 1h4.3c.6 0 1-.4 1-1s-.4-1-1-1h-4.3zM5272.7 2413.7h-4.3c-.6 0-1 .4-1 1s.4 1 1 1h4.3c.6 0 1-.4 1-1s-.4-1-1-1zM5272.7 2417h-4.3c-.6 0-1 .4-1 1s.4 1 1 1h4.3c.6 0 1-.4 1-1 0-.5-.4-1-1-1z"/><path fill="#fff" d="M78.2 13l-8.7 11.7a32.5 32.5 0 11-51.9 25.8c0-10.3 4.7-19.7 12.9-25.8L21.8 13a47 47 0 1056.4 0z"/><path fill="#fff" d="M42.7 2.5h14.6v49.4H42.7z"/></svg>
+                </a>
+            </div>
+            <style type="text/css">

```

Comment:
EEK @nielslyngsoe do we need to inline this CSS, could it not be in an external file. As this makes this XML config file not fun to work with

Reply:
I agree, but the scope of this task was just to change the badge, not the architecture on how its made or configured — I also think that would be a breaking change.